lib/wallet/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -1424,6 +1399,16 @@ assignScriptRedeemers pparams timeTranslation utxo redeemers tx =
                 }
             }
 
+getFeePerByteFromWalletPParams
+    :: ProtocolParameters
+    -> FeePerByte
+getFeePerByteFromWalletPParams pp =
+    let
+        LinearFee LinearFunction {slope}
+            = getFeePolicy $ txParameters pp
+    in
+        FeePerByte $ ceiling slope
```

Comment:
Hi @Anviking, does this not seem a bit simpler to you: (fewer lines, less use of horizontal space)

```suggestion
getFeePerByteFromWalletPParams pp =
    FeePerByte $ ceiling slope
  where
    LinearFee LinearFunction {slope} = getFeePolicy $ txParameters pp
```

Reply:
Done in the other pr